awx/ui_next/src/screens/CredentialType/CredentialTypeList/CredentialTypeListItem.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { string, bool, func } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  DataListAction as _DataListAction,
+  DataListCheck,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  Tooltip,
+} from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import styled from 'styled-components';
+
+import DataListCell from '../../../components/DataListCell';
+import { CredentialType } from '../../../types';
+
+const DataListAction = styled(_DataListAction)`
+  align-items: center;
+  display: grid;
+  grid-gap: 16px;
+  grid-template-columns: 40px;
+`;
+
+function CredentialTypeListItem({
+  credentialType,
+  detailUrl,
+  isSelected,
+  onSelect,
+  i18n,
+}) {
+  const labelId = `check-action-${credentialType.id}`;
+
+  return (
+    <DataListItem
+      key={credentialType.id}
+      aria-labelledby={labelId}
+      id={`${credentialType.id} `}
+    >
+      <DataListItemRow>
+        <DataListCheck
+          id={`select-credential-types-${credentialType.id}`}
+          checked={isSelected}
+          onChange={onSelect}
+          aria-labelledby={labelId}
+        />
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell
+              key="name"
+              aria-label={i18n._(t`credential type name`)}
+            >
+              <Link to={`${detailUrl} `}>
+                <b>{credentialType.name}</b>
+              </Link>
+            </DataListCell>,
+          ]}
+        />
+        <DataListAction
+          aria-label="actions"
+          aria-labelledby={labelId}
+          id={labelId}
+        >
+          {credentialType.summary_fields.user_capabilities.edit ? (
```

Comment:
This is a nit-pick (and I can tell this pattern came from somewhere else because I see it on other lists) but I feel like a conditional render `&&` would be better here than a ternary with an empty string.  So you could do:

```
          {credentialType.summary_fields.user_capabilities.edit && (
            <Tooltip content={i18n._(t`Edit credential type`)} position="top">
              <Button
                aria-label={i18n._(t`Edit credential type`)}
                variant="plain"
                component={Link}
                to={`/credential_types/${credentialType.id}/edit`}
              >
                <PencilAltIcon />
              </Button>
            </Tooltip>
          )}
```

I thought maybe the empty string was necessary for spacing of some kind but I tested it out locally and I don't think so.